website/source/docs/index.html.md

```diff
@@ -3,12 +3,20 @@ layout: "docs"
 page_title: "Documentation"
 sidebar_current: "docs-home"
 description: |-
-  Welcome to the Consul documentation! This documentation is more of a reference guide for all available features and options of Consul. If you're just getting started with Consul, please start with the introduction and getting started guide instead.
+  Welcome to the Consul documentation! This documentation is reference material for all available features and options of Consul. 
 ---
 
 # Consul Documentation
 
-Welcome to the Consul documentation! This documentation is more of a reference
-guide for all available features and options of Consul. If you're just getting
+Welcome to the Consul documentation! This documentation is reference
+material for all available features and options of Consul. 
+
+## Getting Started
+
+If you're just getting
 started with Consul, please start with the
-[introduction and getting started guide](/intro/index.html) instead.
+[Getting Started](https://learn.hashicorp.com/consul/?track=getting-started#getting-started) learning path instead.
+
+## Guided Walkthroughs
+
+Guides are step by step command-line walkthroughs that demonstrate how to perform common operations using Consul, and complement the feature-focused Consul documentation. The guides are located on [HashiCorp Learn](https://learn.hashicorp.com/consul).
```

Comment:
nit: this should probably be broken across multiple lines

Reply:
Done!